File: src/WebClient/WareHouse/src/app/entity/Vendor.ts
import { BaseEntity } from "./BaseEntity";
import { Inward } from "./Inward";
import { WareHouseItem } from "./WareHouseItem";

export interface Vendor extends BaseEntity {
    code: string;
    name: string;
    address: string;
    phone: string;
    email: string;
    contactPerson: string;
    inactive: boolean;
    inwards: Inward[];
    wareHouseItems: WareHouseItem[];
}

File: src/Services/WareHouse/WareHouse.Domain/Entity/Vendor.cs
using System;
using System.Collections.Generic;


#nullable disable

namespace WareHouse.Domain.Entity
{
    public partial class Vendor:BaseEntity
    {
        public Vendor()
        {
            Inwards = new HashSet<Inward>();
            WareHouseItems = new HashSet<WareHouseItem>();
        }

        public string Code { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public string ContactPerson { get; set; }
        public bool Inactive { get; set; }

        public virtual ICollection<Inward> Inwards { get; set; }
        public virtual ICollection<WareHouseItem> WareHouseItems { get; set; }
    }
}
